test/Knp/Snappy/AbstractGeneratorTest.php

```diff
@@ -778,9 +778,13 @@ public function testCheckProcessStatus()
         } catch (\RuntimeException $e) {
             $this->fail('1 status means failure, but no stderr content');
         }
-
-        $this->expectException(\RuntimeException::class);
-        $r->invokeArgs($media, [1, '', 'Could not connect to X', 'the command']);
+        
```

Comment:
StyleCI complains about whitespaces on this line. Could you leave it empty please?